examples/m3.kl

```diff
@@ -0,0 +1,6 @@
+#lang kernel
+
+(import (shift "m2.kl" 1))
+(import "m2.kl")
+
+(example (m2))
```

Comment:
This fails with

    Ambiguous identifier in phase p0 #[m2.kl:9.15-9.17]<m1>
        ScopeSet {_universalScopes = fromList [Scope {scopeNum = 0, scopePurpose = "Root for phase p0"},Scope {scopeNum = 2, scopePurpose = "Module root for /Users/gelisam/working/haskell/klister/examples/m2.kl"},Scope {scopeNum = 3, scopePurpose = "For import at ples/m2.kl:3.1-4.1"},Scope {scopeNum = 9, scopePurpose = "For import at ples/m2.kl:4.1-6.1"}], _phaseScopes = fromList [(Phase {phaseNum = 1},fromList [Scope {scopeNum = 0, scopePurpose = "Root for phase p0"}])]}
        ScopeSet {_universalScopes = fromList [Scope {scopeNum = 0, scopePurpose = "Root for phase p0"},Scope {scopeNum = 2, scopePurpose = "Module root for /Users/gelisam/working/haskell/klister/examples/m2.kl"},Scope {scopeNum = 3, scopePurpose = "For import at ples/m2.kl:3.1-4.1"},Scope {scopeNum = 9, scopePurpose = "For import at ples/m2.kl:4.1-6.1"}], _phaseScopes = fromList [(Phase {phaseNum = 0},fromList [Scope {scopeNum = 0, scopePurpose = "Root for phase p0"}])]}